app/__init__.py

```diff
@@ -65,7 +101,16 @@ def create_app(config_class, database_uri=None):
     db.init_app(app)
     s3.init_app(app)
     compress.init_app(app)
-    talisman.init_app(app, content_security_policy=csp, force_https=force_https)
+    talisman.init_app(
+        app,
+        content_security_policy=csp,
+        force_https=force_https,
+        content_security_policy_nonce_in=[
```

Comment:
what is the purpose of this additional param. documentation, is it so we ignore worker-src and img-src? Either way, why did we need to treat them differently to the rest?

Reply:
Reviewed on call and updated while pairing.

Reply:
Can we just specify script-src since we only refer to `csp_nonce` in the script tags?

Reply:
Set nonces to be used in script-src only 👍